src/index.tsx

```diff
@@ -64,6 +64,11 @@ const SESSION_KEY = 'likecoin_wallet_connector_session';
 
 const WC_BRIGDE = 'https://bridge.walletconnect.org';
 
+const SOCIAL_LOGIN_OPTIONS = {
+  HIDE_SOCIAL: 'likecoin-app-hidesocial',
+  DEFAULT: 'likecoin-app',
```

Comment:
Please also change
`authcoreClientId: options.authcoreClientId || 'likecoin-app',` in line 135

Reply:
[🎨 Set the login social options as a constant](https://github.com/likecoin/likecoin-wallet-connector/pull/60/commits/40d1747944a3a8bc13671a833cfd7f64ee5be0db)